dronekit/__init__.py

```diff
@@ -184,6 +184,13 @@ def __init__(self, north, east, down):
     def __str__(self):
         return "LocationLocal:north=%s,east=%s,down=%s" % (self.north, self.east, self.down)
 
+    def distance(self):
```

Comment:
distance_home is probably a more suitable name; it would be in-line with https://github.com/ArduPilot/pymavlink/blob/master/mavextra.py#L501 for example.